controls/stig_ocp4.yml

```diff
@@ -0,0 +1,1164 @@
+  - audit_rules_login_events_lastlog
+  - audit_rules_login_events_tallylog
+  - audit_rules_sudoers
+  - audit_rules_sudoers_d
+  - audit_rules_usergroup_modification_group
+  - audit_rules_usergroup_modification_gshadow
+  - audit_rules_usergroup_modification_opasswd
+  - audit_rules_usergroup_modification_passwd
+  - audit_rules_usergroup_modification_shadow
+  status: automated
+- id: CNTR-OS-000980
+  levels:
+  - medium
+  title: Red Hat Enterprise Linux CoreOS (RHCOS) must be configured to audit the loading
+    and unloading of dynamic kernel modules.
+  rules:
+  - audit_rules_kernel_module_loading_delete
+  - audit_rules_kernel_module_loading_finit
+  - audit_rules_kernel_module_loading_init
+  - audit_rules_privileged_commands_kmod
+  status: automated
+- id: CNTR-OS-000990
+  levels:
+  - medium
+  title: OpenShift audit records must record user access start and end times.
+  rules:
+  - audit_rules_session_events
+  status: automated
+- id: CNTR-OS-001000
+  levels:
+  - medium
+  title: OpenShift must generate audit records when concurrent logons from different
+    workstations and systems occur.
+  rules:
+  - audit_rules_login_events_faillock
+  - audit_rules_login_events_lastlog
+  status: automated
+- id: CNTR-OS-001010
+  levels:
+  - high
+  title: Red Hat Enterprise Linux CoreOS (RHCOS) must disable SSHD service.
+  rules:
+  - configure_usbguard_auditbackend
+  - kernel_module_usb-storage_disabled
+  - package_usbguard_installed
+  - service_sshd_disabled
+  - service_usbguard_enabled
+  - usbguard_allow_hid_and_hub
+  status: automated
+- id: CNTR-OS-001020
+  levels:
+  - medium
+  title: Red Hat Enterprise Linux CoreOS (RHCOS) must disable USB Storage kernel module.
+  rules:
+  - configure_usbguard_auditbackend
+  - kernel_module_usb-storage_disabled
+  - package_usbguard_installed
+  - service_sshd_disabled
+  - service_usbguard_enabled
+  - usbguard_allow_hid_and_hub
+  status: automated
+- id: CNTR-OS-001030
+  levels:
+  - medium
+  title: Red Hat Enterprise Linux CoreOS (RHCOS) must use USBGuard for hosts that
+    include a USB Controller.
+  rules:
+  - configure_usbguard_auditbackend
+  - kernel_module_usb-storage_disabled
+  - package_usbguard_installed
+  - service_sshd_disabled
+  - service_usbguard_enabled
+  - usbguard_allow_hid_and_hub
+  status: automated
+- id: CNTR-OS-001060
+  levels:
+  - medium
+  title: OpenShift must continuously scan components, containers, and images for vulnerabilities.
+  rules:
+  - container_security_operator_exists
+  status: automated
+- id: CNTR-OS-001080
+  levels:
+  - medium
+  title: OpenShift must use FIPS-validated SHA-2 or higher hash function for digital
+    signature generation and verification (nonlegacy use).
+  rules:
+  - fips_mode_enabled_on_all_nodes
+  status: automated
+- id : needed_rules
+  levels:
+  - medium
+  notes: Manually add rules from SRG-APP-000516-CTR-001325, the SRG is referenced in the published STIG.
+  rules:
```

Comment:
@rhmdnd @Vincent056
The SRG `SRG-APP-000516-CTR-001325` is not referenced in the published STIG.
In the spreadsheets during development it is marked as `Not Applicable (This requirement is NA for this technology)`

In order to keep rule selection the same I have manually added here the rules here.

Reply:
We'll need to update the notes to say that this SRG is *not* referenced in the published STIG.

Reply:
Fixed during the move of rules to the profile file:
https://github.com/ComplianceAsCode/content/pull/11593/commits/5112bcacb9375d27e1505cdc551e02189c0ae20f